Src/GameConnection.hpp

```diff
@@ -3,15 +3,52 @@
 #include <memory>
 #include <vector>
 #include "Object.hpp"
-
+#include "Windows.hpp"
```

Comment:
The presence of the Windows.h include in headers will cause significant compile slow-down to all relevant files.